taxi/admin.py

```diff
@@ -1,34 +1,23 @@
 from django.contrib import admin
-from django.contrib.auth.admin import UserAdmin
 from .models import Driver, Car, Manufacturer
+from django.contrib.auth.admin import UserAdmin
```

Comment:
fix the imports. python modules first, then installed modules, then your own and each group should be separated by blank line